autokeras/blocks/basic.py

```diff
@@ -154,6 +154,48 @@ def build(self, hp, inputs=None):
         return output_node
 
 
+class AttentionBlock(block_module.Block):
+    """The Attention Block. Luong's Multiplicative Style Attention.
+
+    inputs: List of the following tensors:
+      * query: Query `Tensor` of shape `[batch_size, Tq, dim]`.
+      * value: Value `Tensor` of shape `[batch_size, Tv, dim]`.
+      * key: Optional key `Tensor` of shape `[batch_size, Tv, dim]`. If not
+        given, will use `value` for both `key` and `value`, which is the
+        most common case.
+
+    Output shape:
+    Attention outputs of shape `[batch_size, Tq, dim]`.
```

Comment:
I just realized it would be better to put this into the build function's docstring, as # Arguments and # Returns